u2f-gae-demo/war/js/u2fdemo.js

```diff
@@ -214,40 +207,3 @@ function onError(code, enrolling) {
     break;
   }
 }
-
-if (navigator.userAgent.indexOf("iPhone") > -1) {
```

Comment:
can you leave this code in for now so Kaiyu can test and then remove it when it goes into the u2f-api.js file?